ledger/src/blockstore/blockstore_purge.rs

```diff
@@ -160,74 +213,119 @@ impl Blockstore {
             .db
             .batch()
             .expect("Database Error: Failed to get write batch");
+
```

Comment:
nit: I know it was like this before, but can you update the above `.expect()` to use `?` as well